### PR TITLE
Update kitchen config

### PR DESCRIPTION
**Pull request by armiller**

### CODE REVIEW

**Comment by rettigs** [comment]
+1

File: metadata.rb [hunk 1]
```diff
@@ -6,3 +6,6 @@
 long_description 'Installs/Configures omnibus-builder'
 version          '0.1.0'
 
+
+supports         'ubuntu', '12.04'
+supports         'centos', '~> 6'
```
Comment (jordane) [inline metadata.rb]: This needs to be `6.0`

Comment (armiller) [inline metadata.rb]: Is this whats breaking `chef generate`?

Comment (jordane) [inline metadata.rb]: Yes, among other things.

Comment (rettigs) [inline metadata.rb]: So this is an issue with all of the PRs, then?  Should we modify the script to make more PRs that just make this change?

Comment (jordane) [inline metadata.rb]: Yes, probably. I also submitted an upstream patch to use the correct parser for the `supports` method, but even then it won't work until a new version of chef.

Comment (rettigs) [inline metadata.rb]: I'll go ahead and do that now.

Comment (rettigs) [inline metadata.rb]: Looks like @armiller already started it [here](https://github.com/osuosl/chef-repo/commit/67ea4dd69fb75295c695a0118c44bf1cb3b03727).  I'll try and finish it up.

Comment (rettigs) [inline metadata.rb]: [Here's the PR with the finished script](https://github.com/osuosl/chef-repo/pull/257).  I ran it and made all the PRs.

**Comment by jordane** [comment]
+1

**Pull request merged**